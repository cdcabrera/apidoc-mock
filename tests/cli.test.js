const { execSync } = require('child_process');

describe('CLI', () => {
  it('should use default options', () => {
    const output = execSync(`node ./bin/cli.js`);
    expect(output.toString()).toMatchSnapshot('defaults');
  });

  it('should use custom options', () => {
    const output = execSync(`node ./bin/cli.js -p 9000 -d dolorDocs -w lorem -w ipsum`);
    expect(output.toString()).toMatchSnapshot('custom');
  });
});
