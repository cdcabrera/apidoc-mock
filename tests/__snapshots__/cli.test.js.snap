// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI should use custom options: custom 1`] = `"{"port":9000,"dataPath":["lorem","ipsum"],"docsPath":"dolorDocs"}"`;

exports[`CLI should use default options: defaults 1`] = `"{"port":8000,"docsPath":"./.docs"}"`;
